helm/cluster/templates/clusterapi/_helpers_files.tpl

```diff
@@ -61,19 +60,6 @@
 {{- end }}
 {{- end }}
 
-{{- define "cluster.internal.kubeadm.files.kubernetes" }}
```

Comment:
Moved this to control plane specific files, as it's not needed for worker nodes.